Source/Scene/Cesium3DTileset.js

```diff
@@ -1125,11 +1205,19 @@ define([
         return error;
     }
 
+    var scratchCartesian = new Cartesian3();
+
     function computeDistanceToCamera(children, frameState) {
         var length = children.length;
         for (var i = 0; i < length; ++i) {
             var child = children[i];
             child.distanceToCamera = child.distanceToTile(frameState);
+
+            var toCenter = Cartesian3.subtract(child.contentBoundingVolume.boundingVolume.center, frameState.camera.positionWC, scratchCartesian);
+            var distance = Cartesian3.magnitude(toCenter);
+            Cartesian3.divideByScalar(toCenter, distance, toCenter);
+            var dot = Cartesian3.dot(frameState.camera.directionWC, toCenter);
+            child._centerZDepth = distance * dot;
```

Comment:
Yea I think they could. Is `distanceToTile` used elsewhere where changing this would break stuff?